.github/workflows/ci.yaml

```diff
@@ -30,225 +29,249 @@ jobs:
         shell: bash
 
     steps:
-    - name: checkout repository
-      uses: actions/checkout@v2
-
-    - name: install all build dependencies except libtorrent from Ubuntu repos
-      run: |
-        sudo apt update
-        sudo apt install \
-          build-essential cmake git ninja-build pkg-config \
-          libssl-dev libgeoip-dev zlib1g-dev \
-          libboost-dev libboost-chrono-dev libboost-random-dev libboost-system-dev
-      # sudo apt install libqt5svg5-dev qtbase5-dev qttools5-dev # the Qt version in the standard repositories is too old...
-
-    # this will be installed under /opt/qt515. CMake will still find it automatically without additional hints
-    # to speed up the process, only the required components are installed rather than the full qt515-meta-full metapackage
-    - name: install Qt 5.15.2 from an external PPA
-      run: |
-        sudo add-apt-repository ppa:beineri/opt-qt-5.15.2-focal
-        sudo apt install qt515base qt515svg qt515tools
-
-    - name: install libtorrent from source
-      run: |
-        git clone https://github.com/arvidn/libtorrent && cd libtorrent
-        git checkout ${{ env.LIBTORRENT_VERSION_TAG }}
-        cmake -B cmake-build-dir -G "Ninja" \
-          -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-          -Ddeprecated-functions=OFF \
-          --graphviz=cmake-build-dir/target_graph.dot
-        cmake --build cmake-build-dir
-        sudo cmake --install cmake-build-dir --prefix /usr/local
-
-    - name: build qBittorrent
-      run: |
-        cmake -B build -G "Ninja" \
-          -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-          -D${{ matrix.qbt_gui }} \
-          -DVERBOSE_CONFIGURE=ON \
-          --graphviz=build/target_graph.dot
-        cmake --build build
-
-    - name: install qBittorrent
-      run: sudo cmake --install build --prefix /usr/local
-
-    - name: upload artifact as zip
-      uses: actions/upload-artifact@v2
-      with:
-        name: qBittorrent-CI_${{ matrix.os }}-x64_${{ matrix.qbt_gui }}
-        path: |
-          build/compile_commands.json
-          build/install_manifest.txt
-          build/target_graph.dot
-          build/qbittorrent
-          build/qbittorrent-nox
-          libtorrent/cmake-build-dir/compile_commands.json
-          libtorrent/cmake-build-dir/target_graph.dot
-
-  ci_windows:
-    name: Windows + vcpkg
-
+      - name: checkout repository
```

Comment:
Since you have already taken up this task, I would like you to normalize these names as well. Please make them start with a capital letter.

Reply:
Are you sure? by using lowercase we can easily differentiate our steps from predefined steps, for example: "Set up job", "Post checkout repository" from https://github.com/qbittorrent/qBittorrent/runs/3318455194?check_suite_focus=true

Reply:
>by using lowercase we can easily differentiate our steps from predefined steps

Is it worth it? I don't see the point.
It doesn't seem to play a big role in everyday use. And when you need more details, it will not make it difficult to get them.
For me, it just looks illiterate if they start with lowercase, especially if I see that the list has inconsistent elements (especially on the same level).
But if you see a real benefit from their current appearance, forget it. I don't insist.

Reply:
OK, PR updated.